terraform/workspace-variables/review_app_env.yml

```diff
@@ -0,0 +1,21 @@
+---
+AUTHENTICATION_FALLBACK: false
```

Comment:
Maybe we want this environment variable to be set to true so that on review apps we can login

Reply:
Thanks. I've set the value for review to `AUTHENTICATION_FALLBACK: true`